layers/meta-resin-artik/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,8 @@
+do_install_append() {
```

Comment:
Can we add a machine sufix to this? I know we are only using this for artik7 but better safe on long term.